Unaccountable-Magpie/BangazonAPI2
# added annotations to the model's i created

Purpose:
Added annotations to the model's i created for documentation


How it fits in context of the project:
The handbook requires annotations


Specific Feature affected:
Departments, Orders, ProductOrders, and Employees


How to test (Be thorough!):

1. Fetch my branch

2. Look for the annotations at the top of each model listed above

3. Make sure the correct model is referenced
